src/components/DeploymentDetails.vue

```diff
@@ -96,7 +96,7 @@
     }
 
     try {
-      await api.deploymentSchedules.createDeploymentSchedule(props.deployment.id, { active: updatedSchedule.active, schedule: updatedSchedule.schedule })
+      await api.deploymentSchedules.createDeploymentSchedule(props.deployment.id, { active: updatedSchedule.active, schedule: updatedSchedule.schedule, jobVariables: updatedSchedule.jobVariables })
```

Comment:
Any idea why this cannot just be this? 
```suggestion
      await api.deploymentSchedules.createDeploymentSchedule(props.deployment.id, updatedSchedule)
```

Reply:
Yeah there's a _slight_ typing mismatch 
<img width="1006" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/22418768/0eb50bc2-eab6-4a6b-adf1-2ccccb108f9d">

probably why that `if (updatedSchedule.active === null)` check is here. 

I think there's some refactor that could simplify this but didn't get all the way there.